lib/operation.js

```diff
@@ -367,23 +378,43 @@ function median (size) {
  *   .blur(5)
  *   .toBuffer();
  *
- * @param {number} [sigma] a value between 0.3 and 1000 representing the sigma of the Gaussian mask, where `sigma = 1 + radius / 2`.
+ * @param {Object|number|Boolean} [options]
+ * @param {number} [options.sigma] a value between 0.3 and 1000 representing the sigma of the Gaussian mask, where `sigma = 1 + radius / 2`.
+ * @param {string} [options.precision='integer'] How accurate the operation should be, one of: integer, float, approximate.
  * @returns {Sharp}
  * @throws {Error} Invalid parameters
  */
-function blur (sigma) {
-  if (!is.defined(sigma)) {
+function blur (options) {
+  let sigma;
+  if (is.number(options)) {
+    sigma = options;
+  } else if (is.plainObject(options)) {
+    if(!is.number(options.sigma)) {
```

Comment:
@lovell made `options.sigma` required if options object is passed, given that `precision` is only used in `gaussblur`